public/components/ContentPanel/ContentPanelActions.tsx

```diff
@@ -16,19 +16,20 @@
 import React from "react";
 import { EuiButton, EuiFlexGroup, EuiFlexItem } from "@elastic/eui";
 
-interface IndexActionsProps {
-  isAddDisabled: boolean;
-  onClickAdd: () => void;
+interface ContentPanelActionsProps {
+  actions: { text: string; [rest: string]: any }[];
 }
 
-const IndexActions: React.FC<IndexActionsProps> = ({ isAddDisabled, onClickAdd }) => (
+const ContentPanelActions: React.FC<ContentPanelActionsProps> = ({ actions }) => (
   <EuiFlexGroup justifyContent="spaceBetween" alignItems="center">
-    <EuiFlexItem grow={false}>
-      <EuiButton disabled={isAddDisabled} onClick={onClickAdd} data-test-subj="indexActionsAddPolicy">
-        Add policy
-      </EuiButton>
-    </EuiFlexItem>
+    {actions.map(({ text, ...rest }, index) => (
+      <EuiFlexItem grow={false} key={index}>
+        <EuiButton {...rest} data-test-subj={`${text}Button`}>
```

Comment:
Sure, can add button/item props. Was just keeping it simple for now since the current use cases only passed props to button, was going to expand it in the future as the requirements grew.